tests/epyccel/test_epyccel_sets.py

```diff
@@ -114,36 +114,6 @@ def copy_complex():
     assert python_result == pyccel_result
     assert all(isinstance(elem, type(pyccel_result.pop())) for elem in python_result)
 
-def test_Pop_int(language):
-    def Pop_int():
-        se = {2, 4, 9}
-        se.pop()
-        return se
-    epyccel_remove = epyccel(Pop_int, language = language)
-    pyccel_result = epyccel_remove()
-    python_result = Pop_int()
-    assert python_result == pyccel_result
-
-def test_Pop_float(language):
-    def Pop_float():
-        se = {2.7, 4.3, 9.2}
-        se.pop()
-        return se
-    epyccel_remove = epyccel(Pop_float, language = language)
-    pyccel_result = epyccel_remove()
-    python_result = Pop_float()
-    assert python_result == pyccel_result
-
-def test_Pop_complex(language):
-    def Pop_complex():
-        se = {1j, 3j, 6j}
-        se.pop()
-        return se
-    epyccel_remove = epyccel(Pop_complex, language = language)
-    pyccel_result = epyccel_remove()
-    python_result = Pop_complex()
-    assert python_result == pyccel_result
```

Comment:
Is there any reason that these methods were moved? It makes it harder to see what changes you made

Reply:
The reason is to separate pop tests from the other methods to run C and Python tests.